### ISSUE TITLE
Add FCOS variant to use rt-kernel now that PREEMPT_RT is merged into 6.12

### ISSUE THREAD

**Issue by tkarls** [coreos/fedora-coreos-tracker#1843]
### Describe the enhancement

I would like to be able to use FCOS with kernel compiled with the RT patches (PREEMPT_RT) now that this is merged into the mainline kernel (but still require to be compiled with the proper config) to better support latency critical applications.

### System details

This would only be feasible once FCOS starts to ship with kernel 6.12 - I don't expect to have any external patches applied.

**Comment by Nemric** [comment]
I think that it could be related : https://bugzilla.redhat.com/show_bug.cgi?id=2329409

**Comment by travier** [comment]
My understanding is that now that it's merged, once https://bugzilla.redhat.com/show_bug.cgi?id=2329409 is enabled, there will be a single kernel build and the real time part would be enabled with a kernel argument likely, so there would be nothing for us to do here to have it.

**Comment by travier** [comment]
Reading https://www.kernelconfig.io/CONFIG_PREEMPT_RT?q=PREEMPT_RT&kernelversion=6.12.1&arch=x86, maybe it's not the case yet and Fedora will have to build two kernels to get it.

**Comment by tkarls** [comment]
Yes, it is merged into the kernel. But my understanding is that it is a compile time switch and not a boot time switch. So it would require two separate built kernels.

**Comment by LorbusChris** [comment]
This might require a Fedora Change proposal to get the `kernel-rt` RPM built then
